administrator/components/com_actionlogs/src/Helper/ActionlogsHelper.php

```diff
@@ -275,7 +276,7 @@ public static function getContentTypeLink($component, $contentType, $id, $urlVar
     public static function loadActionLogPluginsLanguage()
     {
         $lang = Factory::getLanguage();
-        $db   = Factory::getDbo();
+        $db   =  Factory::getContainer()->get(DatabaseInterface::class);
```

Comment:
```suggestion
        $db   = Factory::getContainer()->get(DatabaseInterface::class);
```
Repeat for the other instances.